codalab/model/tables.py

```diff
@@ -226,5 +236,62 @@
 
   Index('user_user_id_index', 'user_id'),
   Index('user_user_name_index', 'user_name'),
+  UniqueConstraint('user_id', name='uix_1'),
+  sqlite_autoincrement=True,
+)
+
+# Stores verification keys
```

Comment:
This is for the email verification process! Adapted (a simplified version of) this schema from the original Django tables.